xml/containers-orchestration.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+    [
+<!ENTITY % entities SYSTEM "entity-decl.ent">
+%entities;
+]>
+<chapter xml:id="cha-orchestration" xml:lang="en"
+    xmlns="http://docbook.org/ns/docbook" version="5.0"
+    xmlns:xi="http://www.w3.org/2001/XInclude"
+    xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Container Orchestration</title>
+ </info>
+ <sect1 xml:id="sec-pod-deployment-with-podman">
+  <title>Pod Deployment with Podman</title>
+
+  <para>
+   In addition to building and managing images, &podman; makes it possible to
+   work with pods. A pod is a group of one or more containers, with shared
+   storage and network resources. A pod usually encapsulates an application
+   composed of multiple containers into a single unit.
+  </para>
+
+  <para>
+   The <command>podman pod</command> can be used to create, delete, query, and
+   inspect pods. To create a new pod, run the <command>podman pod create --name
+   <replaceable>POD_NAME</replaceable></command> command. This creates a pod
+   with a random name. To list the existing pods, use the <command>podman pod
+   list</command> command. To view a list of running pods, run <command>podman
+   ps -a --pod</command>. Notice that the created pod has an
+   <literal>infra</literal> container identified by the
+   <filename>k8s.gcr.io</filename> name. The purpose of this container is to
+   reserve the namespaces associated with the pod and allow &podman; to add other
+   containers to the pod.
+  </para>
+
+  <para>
+   Using the <command>podman run --pod</command> command, you can run a
+   container and add it to the desired pod. For example, the command below runs
+   a container based on the <literal>suse/sle15</literal> image and adds the
+   container to the <filename>example</filename> pod:
+  </para>
+
+<screen>podman run -dt --pod example registry.suse.com/suse/sle15 bash</screen>
+
+  <para>
+   Containers in a pod can be restarted, stopped, and started without affecting
+   the overall status of the pod. For example, you can stop the container
+   <literal>example</literal> using the <command>sudo podman stop
+   example</command> command.
+  </para>
```

Comment:
I think this is confusing. In the previous example you started a sles container inside of a pod named `example`.

Now you say: you can stop a container named `example` using `<cmd>`; using `example` with two different meanings (first to address a pod, now a single container within a pod) can be confusing IMHO.

I would make something more lengthy such as:

  * When talking about pod creation:
    * We create a pod named `example`
    * We show the actual command being run
    * We show the actual output of `podman ps -a --pod`. We tell the reader the output will look slightly different to him (container IDs, ...)
  * When talking about how to add a container to an existing pod:
    * We show the actual command, we add the container to the `example` pod created before
    * Maybe we don't start an interactive session, we could do something like `sudo podman run -d --pod example registry.suse.com/suse/sle15 sleep 1h`. This start a container that will sleep for 1 hour and then exit
    * We show again the output of `podman ps -a --pod example`
* When talking about how to stop a container:
    * We show the actual command, using the actual data of the "demo" system (aka the IDs shown with the previous `ps -a`)